resources/Microsoft.Windows.Setting.Accessibility/Microsoft.Windows.Setting.Accessibility.psm1

```diff
@@ -328,6 +348,17 @@ class VisualEffect
                 }
                 Set-ItemProperty -Path $global:PersonalizationRegistryPath -Name ([VisualEffect]::TransparencySettingProperty) -Value $transparencyValue 
             }
+            if (0 -ne $this.MessageDurationInSeconds) 
+            {
+                $min = 5
+                $max = 300
+                $messageDurationValue = $this.MessageDurationInSeconds 
+                if ($messageDurationValue -notin 5..300) 

```

Comment:
Can you reuse $min and $max here similar to what you did for the exception message